File: src/main/java/hacker/solution/Dates.java
package hacker.solution;

import java.io.*;

public class Dates {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

        String[] firstMultipleInput = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");

        int month = Integer.parseInt(firstMultipleInput[0]);

        int day = Integer.parseInt(firstMultipleInput[1]);

        int year = Integer.parseInt(firstMultipleInput[2]);

        String res = MyDate.findDay(month, day, year);

        bufferedWriter.write(res);
        bufferedWriter.newLine();

        bufferedReader.close();
        bufferedWriter.close();
    }

}

File: src/main/kotlin/hacker/solutionk/Dates.kt
package hacker.solutionk

import hacker.solutionk.MyDate
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.FileWriter
import java.io.InputStreamReader

fun main(args: Array<String>) {
    val bufferedReader = BufferedReader(InputStreamReader(System.`in`))
    val bufferedWriter = BufferedWriter(FileWriter("dates.txt"))

    val firstMultipleInput =
        bufferedReader.readLine().replace("\\s+$".toRegex(), "").split(" ".toRegex()).dropLastWhile { it.isEmpty() }
            .toTypedArray()

    val month = Integer.parseInt(firstMultipleInput[0])

    val day = Integer.parseInt(firstMultipleInput[1])

    val year = Integer.parseInt(firstMultipleInput[2])

    val res = MyDate.findDay(month, day, year)

    bufferedWriter.write(res)
    bufferedWriter.newLine()

    bufferedReader.close()
    bufferedWriter.close()

}